arroyo/backends/abstract.py

```diff
@@ -1,13 +1,15 @@
 from __future__ import annotations
 
-import logging
+import structlog
 from abc import ABC, abstractmethod, abstractproperty
 from concurrent.futures import Future
 from typing import Callable, Generic, Mapping, Optional, Sequence, Union
+from arroyo.environment import setup_logging
 
 from arroyo.types import BrokerValue, Partition, Topic, TStrategyPayload
 
-logger = logging.getLogger(__name__)
+setup_logging()
```

Comment:
If this call is always required before calling `structlog.get_logger` then I would wrap it in a single helper function.

```
def get_logger(name):
    setup_logging()
    return structlog.get_logger().bind(module=name)
```